### PR TITLE
Docs: Adding code blocks wrapper

### PR DESCRIPTION
**Pull request by Deepshaded**
List of file changed:
- docs/docsite/rst/reference_appendices/general_precedence.rst

##### SUMMARY
- Added code block wrappers to code examples in docs/docsite/rst/reference_appendices/general_precedence.rst
- Fixes : Docs: Add code-block wrappers to code examples in general_precedence.rst #79038

##### ISSUE TYPE
- Docs Pull Request

```

### CODE REVIEW

File: docs/docsite/rst/reference_appendices/general_precedence.rst [hunk 1]
```diff
@@ -47,11 +47,15 @@ When you type something directly at the command line, you may feel that your han
 
 You can override all other settings from all other sources in all other precedence categories at the command line by  :ref:`general_precedence_extra_vars`, but that is not a command-line option, it is a way of passing a :ref:`variable<general_precedence_variables>`.
 
-At the command line, if you pass multiple values for a parameter that accepts only a single value, the last defined value wins. For example, this :ref:`ad hoc task<intro_adhoc>` will connect as ``carol``, not as ``mike``::
+At the command line, if you pass multiple values for a parameter that accepts only a single value, the last defined value wins. For example, this :ref:`ad hoc task<intro_adhoc>` will connect as ``carol``, not as ``mike``:
+
+.. code:: text
```
Comment (ariordan-redhat) [inline docs/docsite/rst/reference_appendices/general_precedence.rst]:
```suggestion
.. code:: shell
```

File: docs/docsite/rst/reference_appendices/general_precedence.rst [hunk 2]
```diff
@@ -47,11 +47,15 @@ When you type something directly at the command line, you may feel that your han
...
 
       ansible -u mike -m ping myhost -u carol
 
-Some parameters allow multiple values. In this case, Ansible will append all values from the hosts listed in inventory files inventory1 and inventory2::
+Some parameters allow multiple values. In this case, Ansible will append all values from the hosts listed in inventory files inventory1 and inventory2:
+
+.. code:: text
```
Comment (ariordan-redhat) [inline docs/docsite/rst/reference_appendices/general_precedence.rst]:
```suggestion
.. code:: shell
```

File: docs/docsite/rst/reference_appendices/general_precedence.rst [hunk 3]
```diff
@@ -133,7 +141,9 @@ Variable values associated directly with a host or group, including variables de
 Using ``-e`` extra variables at the command line
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To override all other settings in all other categories, you can use extra variables: ``--extra-vars`` or ``-e`` at the command line. Values passed with ``-e`` are variables, not command-line options, and they will override configuration settings, command-line options, and playbook keywords as well as variables set elsewhere. For example, this task will connect as ``brian`` not as ``carol``::
+To override all other settings in all other categories, you can use extra variables: ``--extra-vars`` or ``-e`` at the command line. Values passed with ``-e`` are variables, not command-line options, and they will override configuration settings, command-line options, and playbook keywords as well as variables set elsewhere. For example, this task will connect as ``brian`` not as ``carol``:
+
+.. code:: text
```
Comment (ariordan-redhat) [inline docs/docsite/rst/reference_appendices/general_precedence.rst]:
```suggestion
.. code:: shell
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
@Deepshaded 

The build looks good.
Could you please change some lexers?
Thank you!

**Comment by Deepshaded** [comment]
Yes, sure. Changing the highlighted lexers.

**Review by ariordan-redhat — APPROVED** [review]
Looks good to me

**Comment by ariordan-redhat** [comment]
Thank you for contributing to the Ansible docs, @Deepshaded !

**Pull request merged**